README.md

```diff
@@ -22,6 +22,8 @@ test modules that are vendored in a controlrepository or a monorepository. See
 Below is an annotated example Gemfile which should provide a working environment. It's the minimum needed.
 
 ```ruby
+source ENV['GEM_SOURCE'] || 'https://rubygems.org'
```

Comment:
Technically `GEM_SOURCE` is not needed so I'm debating if we need to document it. This isn't about documenting how to write `Gemfile`, rather what we need to be there for gha-puppet to work.

So I'd prefer to keep it simple (as `It's the minimum needed` on line 22 says):
```suggestion
source 'https://rubygems.org'
```